tests/probes_test.go

```diff
@@ -173,18 +167,20 @@ var _ = Describe("[ref_id:1182]Probes", func() {
 			table.Entry("[test_id:1218][posneg:negative]with working HTTP probe and no running server", httpProbe),
 		)
 
-		table.DescribeTable("should not fail the VMI when probing with an IPv6 address", func(livenessProbe *v12.Probe) {
-			libnet.SkipWhenNotDualStackCluster(virtClient)
+		table.DescribeTable("should not fail the VMI when probing with both IPv6 and IPv4 addresses", func(livenessProbe *v12.Probe, IPFamily v1.IPFamily) {
```

Comment:
same here; you should do this on https://github.com/kubevirt/kubevirt/pull/4341/files#diff-f2964358d14f79c439288df507310df44dbc0de205c1ec0a02a78f9d4cbe7c13R137 instead.

Reply:
Done. Removed the whole create-server-on-vmi part.